src/server/system_services/stats_aggregator.js

```diff
@@ -1355,23 +1363,29 @@ function _new_namespace_nsfs_stats() {
 }
 
 // Will return the current nsfs_io_counters and reset it.
-function get_nsfs_io_stats() {
+function get_nsfs_io_stats(only_current = true) {
```

Comment:
what this only_current variable means? not sure it's understood, maybe worth having more informative name

Reply:
updated with `reset_nsfs_counters`.